docs/src/modules/java/pages/actions.adoc

```diff
@@ -187,4 +187,6 @@ Calling an action method through the test kit gives us back an link:{attachments
 +
 * `reply` returns the reply message passed to `effects.reply()` or throws an exception failing the test, if the effect returned was not a reply.
 * `errorDescription` returns the error description when `effects().error()` was returned to signal an error.
-* `forwardedTo` returns details about what message was forwarded and where the call was forwarded (since it is a unit test the forward is not actually executed).
\ No newline at end of file
+* `forwardedTo` returns details about what message was forwarded and where the call was forwarded (since it is a unit test the forward is not actually executed).
++
+By default the integration and unit test are both invoked by `sbt test`. To only run unit tests run `sbt -DonlyUnitTest test` or set up that paremeter to `true` in the sbt console by `set onlyUnitTest := true` and then run `sbt test`
```

Comment:
I'm not sure this applies to Actions. They only have unit tests, right?

Reply:
I think we should mention this here anyway. At some point we should have integration tests for actions, if we don't have it already. And, there are probably other integration tests in the project that you don't want to run when developing the unit tests of actions.